lib/gcov/Makefile.uk

```diff
@@ -0,0 +1,19 @@
+$(eval $(call addlib_s,libgcov,$(CONFIG_LIBGCOV)))
+
+ifeq ($(CONFIG_LIBGCOV),y)
+$(call error_if_gcc_version_lt,13,0)
+endif # CONFIG_LIBGCOV
+
+CINCLUDES-$(CONFIG_LIBGCOV)	+= -I$(LIBGCOV_BASE)/include
```

Comment:
Sorry I missed that earlier, but you also need to add here
```diff
+CXXINCLUDES-$(CONFIG_LIBGCOV)  += -I$(LIBGCOV_BASE)/include
```
along with guards for C++ linkage in `gcov.h`:
```diff
+#ifdef __cplusplus
+extern "C" {
+#endif
... 
+#ifdef __cplusplus
+}
+#endif
```
You can have a look at how we do it in other libraries too.